frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -22,7 +22,7 @@ type IProps = {
 export default ({ className }: IProps) => (<svg width="19" height="19" viewBox="0 0 19 19" fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
 	<g id="align-o">
 		<path className="primary" id="Vector 155 (Stroke)" fillRule="evenodd" clipRule="evenodd" d="M0.170824 7.9752C0.34557 7.67253 0.732592 7.56883 1.03526 7.74358L2.07127 8.34172L1.43846 9.43778L0.402449 8.83964C0.0997799 8.6649 -0.00392238 8.27787 0.170824 7.9752ZM4.60404 11.2654L2.53202 10.0692L3.16484 8.97309L5.23685 10.1694L4.60404 11.2654ZM7.76963 13.0931L5.69761 11.8968L6.33042 10.8007L8.40244 11.997L7.76963 13.0931ZM10.9352 14.9207L8.86319 13.7245L9.496 12.6284L11.568 13.8247L10.9352 14.9207ZM14.1008 16.7484L12.0288 15.5521L12.6616 14.456L14.7336 15.6523L14.1008 16.7484ZM16.2304 17.9779L15.1944 17.3798L15.8272 16.2837L16.8632 16.8818C17.1658 17.0566 17.2696 17.4436 17.0948 17.7463C16.9201 18.0489 16.533 18.1526 16.2304 17.9779Z" fill="currentColor"/>
-		<path className="highlight" id="Rectangle 3623" d="M6.5588 9.46249C5.95347 9.113 5.74606 8.33895 6.09555 7.73361L9.59292 1.67599C9.94242 1.07065 10.7165 0.863249 11.3218 1.21274L17.3794 4.71011C17.9848 5.0596 18.1922 5.83365 17.8427 6.43899L14.3453 12.4966C13.9958 13.1019 13.2218 13.3094 12.6164 12.9599L6.5588 9.46249Z" fill="currentColor"/>
+		<path className="primary" id="Rectangle 3623" d="M6.5588 9.46249C5.95347 9.113 5.74606 8.33895 6.09555 7.73361L9.59292 1.67599C9.94242 1.07065 10.7165 0.863249 11.3218 1.21274L17.3794 4.71011C17.9848 5.0596 18.1922 5.83365 17.8427 6.43899L14.3453 12.4966C13.9958 13.1019 13.2218 13.3094 12.6164 12.9599L6.5588 9.46249Z" fill="currentColor"/>
```

Comment:
If we are now just using one colour get rid of classname entirely as the code implies more complexity by having to check a non existing class.   Same for all svgs.